name.abuchen.portfolio/src/name/abuchen/portfolio/money/currencies.properties

```diff
@@ -97,7 +94,7 @@ BRL.symbol = R$
 
 BSD = Bahamian dollar
 
-BSD.symbol = B$
+BSD.symbol = Bh$
```

Comment:
According to Wikipedia, it is just $

<img width="770" alt="Bildschirmfoto 2025-04-06 um 17 50 18" src="https://github.com/user-attachments/assets/556908da-a8e3-491f-a1d6-ba87f95756de" />